apps/mobile/app/dashboard/(tabs)/settings.tsx

```diff
@@ -24,6 +24,23 @@ export default function Dashboard() {
   const imageQualityMin = useSharedValue(0);
   const imageQualityMax = useSharedValue(100);
 
+  const {
+    data: userSettings,
+    isLoading: isUserSettingsLoading,
+    error: userSettingsError,
+  } = api.users.settings.useQuery(undefined, {
+    /**
+     * Don't retry if the endpoint doesn't exist
+     * maybe we can remove this after x months after release lol
+     */
+    retry: false,
```

Comment:
The settings endpoint was added in 0.25 and it's been more than a month, I think it's fair to assume that everyone should have it running by now.